code/modules/projectiles/ammunition/ballistic/pistol.dm

```diff
@@ -1,4 +1,4 @@
-// 10mm (Stechkin)
+// 10mm (M92)

```

Comment:
This is simply going to confuse players, the pistol still holds the previous name, and `M92` is pretty much a generic factory name for the thing.